setup.py

```diff
@@ -47,17 +51,21 @@ def iter_vendored_files():
         name='ptvsd',
         version=versioneer.get_version(),
         description='Remote debugging server for Python support in Visual Studio and Visual Studio Code', # noqa
-        #long_description=open('DESCRIPTION.md').read(),
-        #long_description_content_type='text/markdown',
+        long_description=long_description,
+        long_description_content_type='text/markdown',
         license='MIT',
         author='Microsoft Corporation',
         author_email='ptvshelp@microsoft.com',
         url='https://aka.ms/ptvs',
```

Comment:
I would add the following to match the classifiers:
```python
python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*;!=3.4.*",
```
This way pip won't try to install on any unsupported version of Python.